src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -97,7 +97,11 @@ describe('Test suite for dictionary concept actions', () => {
       const request = moxios.requests.mostRecent();
       request.respondWith({ status: 200, response: expectedPosts });
     });
-    await fetchSourceConcepts('source', 'query');
+    try {
+      await fetchSourceConcepts('source', 'query');
+    } catch (error) {
+      expect(error).toEqual([]);
```

Comment:
So what happens if the error is not thrown? Will line 103 get executed?

Reply:
Yes if an error happens, line 103 is executed.

Reply:
Did you understand what i said?

Reply:
Sorry, no it will not get executed.

Reply:
We should have tests that get consistently executed every time we run them.